doc/nobol/src/main.rs

```diff
@@ -0,0 +1,148 @@
+#[macro_use]
+extern crate lalrpop_util;
+
+lalrpop_mod!(pub nobol1); // syntesized by LALRPOP
+
+#[derive(PartialEq, Debug)]
+pub struct Var(char);
+
+#[derive(PartialEq, Debug)]
+pub struct Lit(String);
+
+#[derive(PartialEq, Debug)]
+pub struct Stmt(Var, Lit);
```

Comment:
Nitpick: I think `Stmt` is quite generic. What about `Assign` ?

Reply:
ugh, you may be right, though I fear all the ways that name leaks into the rest of the source snippets that are in the text... I'll look into it.

Reply:
I'll go with `Eql`; we're not implementing an interpreter here and that way all the non-terminal names have three characters each.

Reply:
Addressed by b84c123085fd321d0704e0b103ae960424bf68e4